app/models/application_record.rb

```diff
@@ -0,0 +1,4 @@
+# Base ApplicationRecord Class
+class ApplicationRecord < ActiveRecord::Base
+  self.abstract_class = true
+end
```

Comment:
We don't currently use `ApplicationRecord` so please don't introduce it like this - just make your models inherit in the same way as existing ones.

If you want to introduce `ApplicationRecord` then we can discuss that but it should be done as a separate task and introduced globally for consistency.